docs/src/pages/index.module.css

```diff
@@ -21,3 +21,18 @@
   align-items: center;
   justify-content: center;
 }
+
+.button {
+  background-color: #1A176B; 
+  color: white;
+}
```

Comment:
I would do this in a seperate css class in the custom.css file. but i guess can ignore

Reply:
So i would name it `.button--eigen` and put the colors inside that. and on the button itself, i would put `button--eigen` instead of `button--secondary`

Reply:
Done [f1fca46](https://github.com/NethermindEth/el-node-spec/pull/44/commits/f1fca4681b47337eae6c674ee745bfff7187a8a1)